lte/gateway/c/oai/lib/openflow/controller/GTPApplication.cpp

```diff
@@ -265,7 +265,7 @@ static void add_ded_brr_dl_match(
       downlink_fm.add_oxm_field(ipv4_src);
     }
   } else {
-    of13::EthType ip_type(0x086DD);
+    of13::EthType ip_type(0x86DD);
```

Comment:
why is this reverted?

Reply:
It should be 0x86DD and not 0x086DD. 0 is redundant